# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pgadmin3/macro_changes.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/main/events.cpp.orig	2006-11-30 12:48:30.000000000 +0200
+++ ./src/main/events.cpp	2006-12-15 16:27:39.932921500 +0200
@@ -71,7 +71,7 @@
     EVT_STC_UPDATEUI(CTL_SQLPANE,           frmMain::OnPositionStc)
     EVT_CLOSE(                              frmMain::OnClose)
 
-    EVT_AUI_PANECLOSE(                      frmMain::OnAuiUpdate)
+    EVT_AUI_PANE_CLOSE(                      frmMain::OnAuiUpdate)
 
 #ifdef __WXGTK__
     EVT_TREE_KEY_DOWN(CTL_BROWSER,          frmMain::OnTreeKeyDown)
--- ./src/frm/frmQuery.cpp.orig	2006-11-30 12:48:30.000000000 +0200
+++ ./src/frm/frmQuery.cpp	2006-12-15 16:01:56.464460750 +0200
@@ -103,7 +103,7 @@
     EVT_ACTIVATE(                   frmQuery::OnActivate)
     EVT_STC_MODIFIED(CTL_SQLQUERY,  frmQuery::OnChangeStc)
     EVT_STC_UPDATEUI(CTL_SQLQUERY,  frmQuery::OnPositionStc)
-    EVT_AUI_PANECLOSE(              frmQuery::OnAuiUpdate)
+    EVT_AUI_PANE_CLOSE(              frmQuery::OnAuiUpdate)
 END_EVENT_TABLE()
 
 frmQuery::frmQuery(frmMain *form, const wxString& _title, pgConn *_conn, const wxString& query)
--- ./src/frm/frmEditGrid.cpp.orig	2006-11-30 12:48:30.000000000 +0200
+++ ./src/frm/frmEditGrid.cpp	2006-12-15 15:58:29.891550750 +0200
@@ -75,7 +75,7 @@
     EVT_GRID_EDITOR_SHOWN(      frmEditGrid::OnEditorShown)
     EVT_GRID_EDITOR_HIDDEN(     frmEditGrid::OnEditorHidden)
     EVT_GRID_LABEL_RIGHT_CLICK( frmEditGrid::OnLabelRightClick)
-    EVT_AUI_PANEBUTTON(         frmEditGrid::OnAuiUpdate)
+    EVT_AUI_PANE_BUTTON(         frmEditGrid::OnAuiUpdate)
 END_EVENT_TABLE()
 
 
